test/bugs-reported/bug-descriptions

```diff
@@ -0,0 +1,143 @@
+*****************************************
```

Comment:
Removed bug-descriptions. Placed all bugs in test/users/rohanbadlani/bugs. Removed bug-descriptions/